index.html

```diff
@@ -16,7 +16,31 @@
   </head>
 
   <body>
-    <h1 class="text-center">Tailwind Cards Challenge</h1><br>
-    <p class="text-center">Check the README for instructions.</p>
+    <section class="flex flex-wrap mx-auto max-w-[500px] md:max-w-[752px] lg:max-w-[1023px]">
+      <div class="border border-gray-300 rounded-lg m-2 lg:max-w-[325px] md:max-w-[360px] max-w-[500px] hover:shadow-lg">
+        <img src="https://picsum.photos/500/300?v=1" alt="random image" class="rounded-t-lg">
```

Comment:
Try using the [aspect-ratio ](https://tailwindcss.com/docs/aspect-ratio) class to avoid content jumping around the page when reloading it.